src/utils.js

```diff
@@ -53,6 +53,7 @@ const altWidgetMap = {
   },
   array: {
     checkboxes: CheckboxesWidget,
+    hidden: HiddenWidget
```

Comment:
I'm not sure this was needed here... but might be if my implementation is not correct